source/DamageProfile.cpp

```diff
@@ -128,13 +128,13 @@ void DamageProfile::PopulateDamage(DamageDealt &damage, const Ship &ship) const
 			- attributes.Get("piercing resistance")));
 		double highPermeability = attributes.Get("high shield permeability");
 		double lowPermeability = attributes.Get("low shield permeability");
-		double permeability = 0.;
+		double permeability = ship.IsCloaked() ? attributes.Get("cloaked shield permeability") : 0.;
```

Comment:
```suggestion
		double permeability = ship.Cloaking() * attributes.Get("cloaked shield permeability");
```
I wonder if this would make sense... It behaves the same when fully cloaked or fully uncloaked, but adds a transitional period when the player is cloaking/uncloaking.

Reply:
I didn't know that was possible, but I think it makes more sense.